build/target-repository/docs/rector_rules_overview.md

```diff
@@ -14,7 +14,7 @@
 
 - [EarlyReturn](#earlyreturn) (9)
 
-- [Instanceof_](#instanceof) (1)
+- [Instanceof_](#instanceof_) (1)
```

Comment:
this probably bug on rule-doc-generator, the `rector_rules_overview.md` is generated via :

```
composer docs
```

Feel free to send a patch to https://github.com/symplify/rule-doc-generator/ instead